### ISSUE TITLE
MLI Package with cmake configuration

### ISSUE THREAD

**Issue by njsyw1997** [hypre-space/hypre#756]
Hi, I am trying to enable the original smoothed aggregation method in MLI package. The manual shows it can be used with the IJ interface as a black-box method. How can I do this with cmake? Seems `HYPRE_LSI_MLISetParams` comes from the FEI interface and does not support cmake configuration.

**Comment by zongy17** [comment]
Yes. I also want to use smoothed aggregation AMG in IJ interface, and got the same problem. Hope to get some advice soon. I try turning on the option "hypre_using_fei=yes" in _configure_ file manually, but it led to errors when compiling: 

> make[2]: *** No rule to make target `/fei_Data.hpp', needed by `HYPRE_LSI_Dsuperlu.o'.  Stop.

I don't know how to make this thing work?

**Comment by rfalgout** [comment]
The FEI interface will eventually be removed from hypre, so we don't plan to add CMake support to build it.  Also, the MLI solver in hypre is not being actively supported.  Smooothed aggregation is Sandia's main approach for doing AMG so I would recommend looking at MueLu in Trilinos.

**Comment by zongy17** [comment]
Hi, thanks for your reply @rfalgout .
By the way, I am also very interested in SSAMG in hypre after reading the paper (https://arxiv.org/abs/2205.14273). I want to reproduce the numerical experiments in the paper. I notice that you are the second author, and I wonder if you could provide the code files that set up the problems and invoke SSAMG with appropriate APIs?
